inference/inference.py

```diff
@@ -421,7 +448,13 @@ def autoregressive_inference(params, ic, valid_data_full, model):
                 hour_of_day = date_obj.timetuple().tm_hour
                 hours_since_jan_01_epoch = 24*day_of_year + hour_of_day
                 ics.append(int(hours_since_jan_01_epoch/6))
-        n_ics = len(ics)
+
+    if params.is_log_aggregated_metrics_to_wandb:
```

Comment:
currently this code path is not possible, right? I'd lean to just reverting this change if so.

Reply:
Correct, it's hard coded. I'll revert.